src/lib/project_analyzer.js

```diff
@@ -0,0 +1,21 @@
+const { PROJECT_TYPES } = require('./consts');
+const { ScrapyProjectAnalyzer } = require('./scrapy-wrapper/src/ScrapyProjectAnalyzer');
```

Comment:
do we really need the `src` folder?

Reply:
The template gets downloaded and unzipped next to the `src` folder, but I guess it can be there in a separate folder 👍🏽